js/repl/Repl.js

```diff
@@ -553,12 +553,13 @@ const styles = {
     flex: "0 0 auto",
   }),
   repl: css({
-    height: "100%",
+    height: "calc(100vh - 50px)", // 50px is the header's height
```

Comment:
I'm not sure if `calc` has a good browser support?

Reply:
From IE11+ calc is supported.

Reply:
Is there a way with the emotion library to do something like this?

```css
.repl {
  heigh: 100%;
  height: calc(100vh - 50px);
}
```

Reply:
You mean having two times the `height` attribute? Like a fallback in case the `calc` doesn't work?

We can try to ping @tkh44.

Reply:
> You mean having two times the height attribute? Like a fallback in case the calc doesn't work?

Yes!
Anyway, later I will try to make it work without using `calc`.

Reply:
Seems to work fine 😄 
<img src="https://user-images.githubusercontent.com/662750/35486051-94ac01b4-0425-11e8-9750-f297ac080fbb.png" height="360px"/>

Reply:
Perfect, thanks @tkh44! We need to remove the object and use the template string instead.